assets/css/campfire-games.css

```diff
@@ -0,0 +1,42 @@
+h2 {
+  margin-bottom: 20px;
+}
+
+.container {
+  padding-top: 30px;
+}
+
+#teams-container {
+  background-color: rgb(227,219,203);
+}
+
+.marshie {
+  height: 200px;
+  width: 200px;
+  margin-bottom: 5px;
+}
+
+.row {
+  padding-top: 60px
+}
+
+.main {
+  font-size: 1.3rem;
+}
+
+#footer {
+  padding-top: 100px;
+}
```

Comment:
```suggestion
```
This looks like it's a little too much space on my end--could we just leave it the same spacing as it was on the main page (which should apply if you remove this style)?